### DISCUSSION TITLE
tried updating 1.1f to another version now I have lost everything

### DISCUSSION THREAD

**Discussion by quotelawrence** [arkypita/LaserGRBL#894] [Q&A]
my laser engraver has lost the grbl 1.1f version and is not talking through the computer can someone point me in a direction of restoring the nano uno chip to the original settings as I believe I had done the wrong thing and it is not communicating it will not register a port and it say it cannot find

**Comment by arkypita** [comment]
> it is not communicating it will not register a port and it say it cannot find

The serial port should be detected regardless of what is loaded on the atmega328 chip, because it is managed by serial chip.

First of all: I suggest you to double check your cable, and test with different cables.
Some cable only use to give power, but not transmit data signals.

Second: this is not a problem of LaserGRBL or grbl itself. I suggest you to google for "arduino nano com port not found" or something similar, you'll find plenty of infos about your problem in more arduino generic forums.
